packages/indexer-service/src/query-fees/allocations.ts

```diff
@@ -86,10 +90,13 @@ export class AllocationReceiptManager implements ReceiptManager {
     }
 
     const receipt = this._parseAllocationReceipt(receiptData)
+    console.log('Parsed receipt: ', receipt)
```

Comment:
This should probably be a call to `logger` instead of `console`? Same below on L99.

Reply:
I'll remove these logs.  I actually had them in here to debug something else entirely 😉